woocommerce/woocommerce
# Add a new set of CLI commands to WooCommerce Beta Tester

### All Submissions:


### Changes proposed in this Pull Request:

To make integration between WooCommerce Beta Tester and external systems easier, this exposes a set of commands for installing any live branch that is listed in the live branches manifest.

This also fixes a few minor bugs and formatting issues I found during development.






### How to test the changes in this Pull Request:



1. Check out this branch locally and run this plugin on your local dev environment. Test without WooCommerce plugin installed first.
2. Make sure you turn on the live branches page by adding `$this->register_page()` [here](https://github.com/woocommerce/woocommerce/blob/01f0af240d37bf0ed4f631a9fdd4f0a7fff2b606/plugins/woocommerce-beta-tester/includes/class-wc-beta-tester-live-branches.php#L23).
3. Make sure your dev environment has wp-cli installed. 
3. Start by trying to install a WC live branch that doesn't exist: `wp wc-beta-tester install a/fake-branch` This should inform you that there is no such branch to install
4. Now try install a branch that does exist e.g. cot-main: `wp wc-beta-tester install cot-main`. This should work and inform you that cot-main is installed
5. Now activate it. Run `wp wc-beta-tester activate cot-main`. This should succeed
6. Now go to the live branches page on your admin panel (Live Branches under WooCommerce menu). You should see the branch you chose is installed and active as listed in the info at top of page.



### Other information:




### FOR PR REVIEWER ONLY:
